includes/admin/upgrades/upgrade-functions.php

```diff
@@ -1291,7 +1299,7 @@ function give_v20_logs_upgrades_callback() {
 
 		while ( $forms->have_posts() ) {
 			$forms->the_post();
-			global $post;
+			global $post, $wpdb;
```

Comment:
@mehul0810 I saw `$wpdb` defined multiple times in this function and also do not define it inside `while` loop.